server/sonar-server/src/main/java/org/sonar/server/ce/ws/TaskAction.java

```diff
@@ -103,4 +118,40 @@ private void checkPermission(@Nullable String projectUuid) {
       throw insufficientPrivilegesException();
     }
   }
+
+  private static void maskErrorStacktrace(Request wsRequest, CeActivityDto ceActivityDto) {
+    Set<AdditionalField> fromRequest = TaskAction.AdditionalField.getFromRequest(wsRequest);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Convert this Set to an EnumSet. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1641)